source/parser.hera

```diff
@@ -1693,7 +1693,10 @@ BinaryOpSymbol
   "<="
   ">="
   "<<"
-  "<"
+  # NOTE: Avoid matching JSX opening tag by requiring non-identifier character
+  # (e.g. whitespace) after "<", or no whitespace before "<" (for e.g. "1<2").
+  /<(?!\p{ID_Start}|[_$])|(?<=\S)</ ->
```

Comment:
Lookbehinds don't work in Safari so we may need to find another way.

Reply:
Are we OK forbidding `1<2` and `x<y`?

Alternatively, inside the `->` can we get the text right before the match, and return `$skip` if appropriate?

I was thinking about this but wasn't sure we needed the Civet compiler to run in Safari. Might be nice though.

Reply:
We can require mandatory whitespace between binary operators.

The compiler should be able to run in Safari (Civet Playground etc.)